server/sonar-server/src/test/java/org/sonar/server/issue/IssueServiceMediumTest.java

```diff
@@ -94,21 +94,11 @@ private void index() {
   }
 
   @Test
-  public void get_by_key() {
-    RuleDto rule = newRule();
-    ComponentDto project = newProject();
-    ComponentDto file = newFile(project);
-    IssueDto issue = saveIssue(IssueTesting.newDto(rule, file, project));
-
-    assertThat(service.getByKey(issue.getKey())).isNotNull();
-  }
-
-  @Test
   public void assign() {
     RuleDto rule = newRule();
     ComponentDto project = newProject();
     ComponentDto file = newFile(project);
-    userSessionRule.login("john");
+    userSessionRule.login("john").addProjectUuidPermissions(UserRole.USER, project.uuid());
```

Comment:
FYI, adding addProjectUuidPermissions is now required because checks are bases on DB and no more on ES.